mne/decoding/search_light.py

```diff
@@ -273,7 +279,7 @@ def score(self, X, y):
         # For predictions/transforms the parallelization is across the data and
         # not across the estimators to avoid memory load.
         parallel, p_func, n_jobs = parallel_func(
-            _sl_score, self.n_jobs, max_jobs=X.shape[-1])
+            _sl_score, self.n_jobs, max_jobs=X.shape[-1], verbose=False)
```

Comment:
Is this necessary? I feel like this might already do something reasonable if you do `verbose=_check_verbose(self.verbose)` or whatever. IIRC I made it so that if you have 4 jobs you get 4 progressbars or something...

Reply:
This is to silence joblib, everywhere else in this module `parallel_func` is used with `verbose=False`, I don't think it affects progressbars.

Reply:
Oh yeah we don't want that